app/controllers/evaluators_controller.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
-class ManageEvaluatorsController < ApplicationController
-  include ManageEvaluatorsHelper
-
+class EvaluatorsController < ApplicationController
   before_action -> { authorize_user('challenge_manager') }
-  before_action :set_phase, only: [:index, :create, :destroy]
+
+  # All routes are scoped to a Challenge Phase
+  before_action :set_challenge_phase
 
   def index
-    handle_existing_phases
+    @evaluator_invitations = @phase.evaluator_invitations
+    @existing_evaluators = @phase.evaluators
```

Comment:
I think it's easier to read directly inside the action when there is some specific data needed